db/setup/storage-download/storage-download.py

```diff
@@ -0,0 +1,52 @@
+import os
+from azure.storage.blob import BlobServiceClient
+
+
+def folder_processing(storage_url, container_name):
+    """
+    This function downloads all the files from a container in a storage account
+    to the local directory "test"
+
+    This serves as a way to locally download the container files for processing and importing within the db
+
+    Parameters:
+    - storage_url: the url of the storage account
+    - container_name: the name of the container
+
+    Returns: None
+    """
+
+    # Create a blob service client
+    blob_service_client = BlobServiceClient.from_connection_string(conn_str=storage_url)
```

Comment:
don't create clients within lower-level functions; pass the client as a parameter to the function.

this will allow you to pass mocks to the function during testing.

Reply:
I've updated this file to make it more usable